src/modules/projects/components/tables/ProjectClientEnrollmentsTable.tsx

```diff
@@ -146,6 +148,15 @@ export const ENROLLMENT_COLUMNS: {
       return null;
     },
   },
+  assignedStaff: {
+    ...ASSIGNED_STAFF_COL,
+    render: (e) => {
+      if ('household' in e && isRenderFunction(ASSIGNED_STAFF_COL.render)) {
```

Comment:
So it looks like isRenderFunction casts ASSIGNED_STAFF_COL.render to be `(e:any)=> React.ReactNode`. This seems like an indirect way of just casting e to `any`.

Suggest we either cast e in a more obvious way, or use a guard function. So:

```ts
    render: (e:any) => {
      if ('household' in e){
        return ASSIGNED_STAFF_COL.render(e.household);
      }
      return null;
    },
```

or


```ts
//guard function
const isHouseholdWithStaff = (e: any): e is {household: HouseholdWithStaffAssignmentsFragment} => {
  return 'household' in e && 'staffAssignments' in e.household
}
// ....

    render: (e) => {
      return isHouseholdWithStaff(e) ? ASSIGNED_STAFF_COL.render(e.household) : null
    }
```

Reply:
I like the idea of a guard function. Thanks for the suggestion @ttoomey !